src/components/MainView.js

```diff
@@ -18,18 +18,16 @@ function MainView({ children }) {
   const { below } = useViewport()
   const connectedGarden = useConnectedGarden()
   const [openPreferences, closePreferences, preferenceOption] = usePreferences()
-  const mobileMode = below('medium')
-  const compactMode = below('large')
-  const [showSidebar, setShowSidebar] = useState(!mobileMode)
+  const [showMobileSidebar, setShowMobileSidebar] = useState(false)
   const [createProposalModalVisible, setCreateProposalModalVisible] = useState(
     false
   )
-  let loadingGardenState = true
 
   const handleToggleSidebar = useCallback(() => {
-    setShowSidebar(prevShowSidebar => (!mobileMode ? true : !prevShowSidebar))
-  }, [mobileMode])
+    setShowMobileSidebar(prevShowMobileSidebar => !prevShowMobileSidebar)
```

Comment:
Updated this logic as we only toggle the mobile sidebar when we are in mobile mode, so don´t think the condition is necessary